lascli/parser/predictions.py

```diff
@@ -23,6 +23,7 @@ def create_predictions_parser(subparsers):
     create_predicton_parser.add_argument('--training-id', help='Id of training to use for predictions')
     create_predicton_parser.add_argument('--auto-rotate', action='store_true', default=False)
     create_predicton_parser.add_argument('--max-pages', type=int, default=1)
+    create_predicton_parser.add_argument('--rotation', type=int, default=None)
     create_predicton_parser.add_argument('--image-quality', type=str, default='LOW', choices={'LOW', 'HIGH'})
```

Comment:
Can you make sure these are not sent to the API if a value is not explicitly provided? We usually want the default model pre/postprocess config to kick in when user is omitting any of these values

Reply:
That is handled by the dictstrip method in the python SDK, or do you mean image-quality and max_pages?